src/routes/safe/components/Balances/Coins/index.tsx

```diff
@@ -68,10 +69,25 @@ const Coins = (props: Props): React.ReactElement => {
     trackEvent({ category: SAFE_NAVIGATION_EVENT, action: 'Coins' })
   }, [trackEvent])
 
-  React.useMemo(() => {
+  useMemo(() => {
     setFilteredData(getBalanceData(activeTokens, selectedCurrency, currencyValues, currencyRate))
   }, [activeTokens, selectedCurrency, currencyValues, currencyRate])
 
+  const showCurrencyTooltip = (valueWithCurrency: string, balanceWithSymbol: string): React.ReactElement | null => {
```

Comment:
Please don't define what means to be a component inside another component's body as it will be recreated every render and possible mounted/unmounted

Reply:
https://twitter.com/0xca0a/status/1269562072725098501